modules/utxo-lib/src/bitgo/UtxoPsbt.ts

```diff
@@ -35,6 +41,9 @@ export interface PsbtOpts {
   network: Network;
 }
 
+// TODO: upstream does `checkInputsForPartialSigs` before doing things like
+// `setVersion`. Our inputs could have tapscriptsigs (or in future tapkeysigs)
+// and not fail that check. Do we want to do anything about that?
```

Comment:
What happens if the inputs have partial signatures in upstream?

Reply:
It prevents modifying the PSBT if it would invalidate any sigs.